packages/lib/src/components/Trustly/Trustly.tsx

```diff
@@ -0,0 +1,42 @@
+import { h } from 'preact';
+import CoreProvider from '../../core/Context/CoreProvider';
+import RedirectElement from '../Redirect';
+import RedirectButton from '../internal/RedirectButton';
+import './Trustly.scss';
+
+class TrustlyElement extends RedirectElement {
+    public static type = 'trustly';
+
+    get displayName() {
+        return this.props.name || this.constructor['type'];
+    }
+
+    render() {
+        if (this.props.showPayButton) {
```

Comment:
We should hide only the pay/redirect button if showPayButton is `false`. In this case, it is hiding the whole Component